src/components/App.tsx

```diff
@@ -434,8 +425,6 @@ class App extends React.Component<ExcalidrawProps, AppState> {
   private onBlur = withBatchedUpdates(() => {
     isHoldingSpace = false;
     this.setState({ isBindingEnabled: true });
-    this.saveDebounced();
-    this.saveDebounced.flush();
```

Comment:
currently, the data is flushed in onBlur, so I am ok with passing onBlur callback.
Thoughts?